examples/nodejs/stock.js

```diff
@@ -2,29 +2,36 @@
 
 var http = require('http');
 
-module.exports = function (context, callback) {    
+module.exports = async function (context) {
     let body = context.request.body;
     console.log(`body text: ${body['text']}`);
 
     var symbol = body['text'].split(' ')[1];
 
-    http.get({
-        host: 'finance.google.com',
-        path: `/finance/info?q=NYSE:${symbol}`
-    }, function(response) {
-        var resp = '';
-        response.on('data', function(d) {
-            resp += d;
-        });
-        response.on('end', function() {
-
-            try {
-                var parsed = JSON.parse(resp.slice(3));
-                var lastTrade = parsed[0]['l_cur']
-                callback(200, `{ "text": "${symbol} last traded at ${lastTrade}" }`);
-            } catch (e) {
-                callback(200, `{ "text": "Error (invalid NYSE symbol?)" }`);
-            }
+    return new Promise(function(resolve, reject) {
+        http.get({
```

Comment:
@soamvasani There are a number of Promise-based HTTP clients available for Node, which would indeed be much cleaner. However, wouldn't adding additional dependencies require a custom fission environment? I assumed you wanted to use the native `http` module to avoid adding dependencies.